src/Bicep.LangServer/Handlers/BicepTextDocumentSyncHandler.cs

```diff
@@ -58,11 +58,10 @@ public override Task<Unit> Handle(DidCloseTextDocumentParams request, Cancellati
             return Unit.Task;
         }
 
-        private static TextDocumentSaveRegistrationOptions GetSaveRegistrationOptions()
-            => new TextDocumentSaveRegistrationOptions
-            {
-                DocumentSelector = DocumentSelectorFactory.Create(),
-                IncludeText = true,
```

Comment:
>IncludeText [](start = 16, length = 11)

There doesn't appear to be an equivalent for `IncludeText`, but nothing appears to be broken.